### ISSUE TITLE
Disable side-by-side

### ISSUE THREAD

**Issue by brandonchinn178** [dlvhdr/diffnav#10]
Is it possible to disable side-by-side diffs? I notice you manually add the argument here: https://github.com/dlvhdr/diffnav/blob/986291b684ce85a97ee23bc75f2668b584c768d5/diffviewer.go#L119-L121

It doesn't seem possible to undo that

**Comment by chrisgrieser** [comment]
I agree. With the file tree taking up space, side-by-side becomes more squished than using other diff viewers, making it not a great default in my view.

**Comment by dlvhdr** [comment]
I will change that but just FYI you can hide the tree with <kbd>e</kbd>

**Comment by samuelroland** [comment]
Thanks for this nice tool ! This will improve my review experience ! 
+1 on this, I'm really used to the default layout of `delta` so this would be really helpful to have an option on start (maybe `--combined` or making combined the default and keep `-s, --side-by-side` like `delta` for the side-by-side mode ?).

**Comment by JaredSharplin** [comment]
Keen for this, just as a personal preference for the unified diff view.

**Comment by raylu** [comment]
would you take a PR for this feature?

(does diffnav take any flags and/or are you interested in having it read a config file?)
